apps/total-typescript/src/templates/tip-template.tsx

```diff
@@ -341,4 +349,33 @@ const ReplyOnTwitter: React.FC<{tweet: string}> = ({tweet}) => {
   )
 }
 
+const SubscribeForm = () => {
+  const router = useRouter()
+  return (
+    <div
+      id="tip"
+      className="2xl:px-0 px-3 w-full md:pt-1 pt-2 pb-3 flex md:flex-row flex-col justify-between items-center gap-5 border-b border-gray-800"
+    >
+      <div className="lg:flex-shrink-0 inline-flex items-center gap-2 lg:text-lg md:text-base text-lg font-medium leading-tight">
+        <div
+          aria-hidden="true"
+          className="flex-shrink-0 rounded-full w-10 h-10 bg-gray-800 flex items-center justify-center"
+        >
+          <MailIcon className="w-5 h-5 text-cyan-300" />
+        </div>{' '}
+        New TypeScript tips delivered to your inbox:
+      </div>
+      <SubscribeToConvertkitForm
+        actionLabel="Subscribe for more tips"
+        onSuccess={(subscriber: any) => {
+          if (subscriber) {
+            const redirectUrl = redirectUrlBuilder(subscriber, '/confirm')
+            router.push(redirectUrl)
+          }
+        }}
```

Comment:
important to track here

https://github.com/skillrecordings/products/blob/3b0643ef869954eca1f9500981dc03fa70503a8e/apps/total-typescript/src/components/exercise-overlay.tsx#L322-L339

I'd just reload with the toast too since they probably don't want to move to a different page